horace_core/sqw/@Experiment/private/combine_experiments_.m

```diff
@@ -1,4 +1,5 @@
 function [exp,nspe] = combine_experiments_(exp_cellarray,allow_equal_headers,keep_runid)
+%COMBINE_EXPEERIMENTS_
```

Comment:
Is this some new or existing convention I was not aware of?

Reply:
I have recently restored this code to the state it was before refactoring

Reply:
I'm leaving this as-is - too many cooks already